apis/flowcollector/v1beta2/flowcollector_types.go

```diff
@@ -373,6 +374,83 @@ type FlowCollectorIPFIXReceiver struct {
 	Transport string `json:"transport,omitempty"`
 }
 
+type FlowCollectorOpenTelemetryLogs struct {
+	// Set `enable` to `true` to send logs to Open Telemetry receiver.
+	//+kubebuilder:default:=true
+	Enable *bool `json:"enable,omitempty"`
+}
+
+type FlowCollectorOpenTelemetryMetrics struct {
+	// Set `enable` to `true` to send metrics to Open Telemetry receiver.
+	//+kubebuilder:default:=true
+	Enable *bool `json:"enable,omitempty"`
+
+	// Prefix added to each metric name
+	// +optional
+	Prefix string `json:"prefix,omitempty"`
+
+	// How often should metrics be sent to collector
+	// +kubebuilder:default:="20s"
+	PushTimeInterval *metav1.Duration `json:"pushTimeInterval,omitempty"`
+
+	// Time duration of no-flow to wait before deleting data item
+	// +kubebuilder:default:="2m"
+	ExpiryTime *metav1.Duration `json:"expiryTime,omitempty"`
```

Comment:
the same setting exists for regular prom metrics but we chose not to expose it as this is more of an implementation detail. I think we should do the same here ;

or if we really want to expose it, then it might be better to set as an advanced setting, and even share the same between prom & otel configs (because at the end of the day this is really the thing: how long do we keep in memory metrics that reference elements that may disappear, such as those having a pod name label and that pod doesn't exist anymore)

But for simplicity I wouldn't even bother about that, and just remove this setting

Reply:
Ok I'll remove it then

Reply:
Should we keep the `pushTimeInterval` ?